package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "node-boleto",
-  "version": "2.2.3",
+  "version": "2.2.4",
```

Talvez vale mudar o numero referente à feature @kayquecoelho 😄 
```suggestion
  "version": "2.3.2",
```